friends/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+// import logo from './logo.svg';
+import axios from 'axios';
+
+import './App.css';
+
+class App extends React.Component {
+constructor() {
+  super();
+  this.state = {
+    friends:[],
+  };
+}
+
+  componentDidMount(){
+    axios
+    .get('http://localhost:5000/friends')
+    .then (response => {this.setState({friends: response.data});})
+    .catch (err => console.log(err));
+    }
+
+  render() {
+    return (
+      <div className="App">
+
+        <h1>Order Of The Besties</h1>
+
+          {this.state.friends.map(friends => (
```

Comment:
Don't forget to make use of components! The instructions state that you should create a separate component, perhaps a FriendsList component, to handle this rendering information.